model/firewall.rb

```diff
@@ -12,14 +12,18 @@ class Firewall < Sequel::Model
   include Authorization::TaggableMethods
   include Authorization::HyperTagMethods
   def hyper_tag_name(project)
-    "project/#{project.ubid}/firewall/#{ubid}"
+    "project/#{project.ubid}/location/#{display_location}/firewall/#{name}"
```

Comment:
You need to update existing hyper tags

Reply:
Yes, I'll be updating them after deploying the first commit and migrating existing firewalls, so it will be a part of migration.